NOAA-EMC/global-workflow
# Feature/ecflow suite generator

**Description**
This PR is to merge in the workflow generator application that uses a YAML file to create suite definition files for ecflow. 

**Type of change**

**How Has This Been Tested?**
To test: 
1. Change into the workflow directory: ` cd global-workflow/workflow_generator`
1. Ensure the YAML file is configured with the suites/tasks that are wanted. 
1. Run the utility: `python3 setup_workflow.py --expdir ../parm/config`

** Remaining items **
1. Additional documentation
1. Code documentation
1. Test suite with existing production ecflow definition file.
